chop/stochastic.py

```diff
@@ -106,17 +106,37 @@ def normalize_gradient(grad, normalization):
     return grad
         
 
-
 class PGD(Optimizer):
-    """Projected Gradient Descent"""
+    """
```

Comment:
I think we should follow Python's docstring convention (https://www.python.org/dev/peps/pep-0257/) . Basically says that docstring should start with a one-line summary ended with a dot and a blank line (so basically start after the """ and remove the ==== block) . Some frameworks like sphinx rely on this convention to provide a summary of the method for instance.